packages/manager/apps/pci-object-storage/src/api/data/download.ts

```diff
@@ -1,7 +1,7 @@
 import { v6 } from '@ovh-ux/manager-core-api';
 import { addWeeks } from 'date-fns';
 import { OPENIO_PRESIGN_EXPIRE } from '@/constants';
-import { TContainer } from '@/pages/objects/container/object/show/Show.page';
+import { TContainer } from '@/pages/dashboard/BucketPropertiesCard';
```

Comment:
API data should never depend on a pure UI component, please reverse the dependency

Reply:
dependency with a UI folder is still here, please remove it